packages/spec-junit-reporter/package.json

```diff
@@ -17,7 +17,7 @@
 	"private": true,
 	"peerDependencies": {
 		"mocha": "^8.1.3",
-		"mocha-junit-reporter": "^2.0.0"
```

Comment:
It's odd that this failed to also update the dependency in `test/e2e` form 2.0.0 to 2.0.2 :thinking: